package.json

```diff
@@ -32,6 +32,10 @@
     "split": "scripts/split.js",
     "start": "node -r esm scripts/start/index.js",
     "test": "yarn lint",
+    "test:e2e:chromium": "npx rimraf ./reports && testcafe chromium",
```

Comment:
Note for me:
As `rimraf` is widely used across several scripts entry we can consider adding it as a devDependency.